tests/Functional/Action/Lti/UpdateLti1p1OutcomeActionTest.php

```diff
@@ -249,4 +249,30 @@ private function getLtiInstance(): LtiInstance
 
         return $ltiInstance;
     }
+
+    private function getValidReplaceResultResponseXml(string $messageIdentifier): string
```

Comment:
suggestion: instead of hardcoding the xml in your test, what about using the [available template](https://github.com/oat-sa/lib-lti1p3-basic-outcome/blob/master/templates/response/replaceResultResponse.xml.twig) for this ? wdyt ?

Reply:
I think it's better to not rely on vendor party templates in tests.

Reply:
This xml stream is coming form frozen IMS specs, meaning it wont change.

But up top you, I can understand your point :)